BetterShardsBungee/src/main/java/vg/civcraft/mc/bettershards/bungee/BetterShardsBungee.java

```diff
@@ -62,11 +62,12 @@ private void loadDB() {
 	public void loadConfiguration() {
 		if (!getDataFolder().exists())
 			getDataFolder().mkdir();
+		
 		configManager = ConfigurationProvider.getProvider(YamlConfiguration.class);
 		if (!configFile.exists()) {
 			try {
                 configFile.createNewFile();
-                try (InputStream is = getResourceAsStream("bungee" + File.separator + "config.yml");
+                try (InputStream is = getResourceAsStream("config.yml");
```

Comment:
Why did you change this? Won't there be duplicate config from bukkit and bungee.